test_scripts/API/SetAudioStreamingIndicator/ideas_SetAudioStreamingIndicator_conditions_for_SDL_to_handle_related_cases.txt

```diff
@@ -0,0 +1,111 @@
+./test_scripts/API/SetAudioStreamingIndicator/043_ATF_N_TC_SetAudioStreamingIndicator_media_app_HMI_replies_IN_USE.lua Check_that_SDL_responds_with_resultcode_IN_USE_from_HMI_incase_SetAudioStreamingIndicator_is_correct
+./test_scripts/API/SetAudioStreamingIndicator/044_ATF_N_TC_SetAudioStreamingIndicator_media_app_HMI_replies_DATA_NOT_AVAILABLE.lua Check_that_SDL_responds_with_resultcode_DATA_NOT_AVAILABLE_from_HMI_incase_SetAudioStreamingIndicator_is_correct
+./test_scripts/API/SetAudioStreamingIndicator/045_ATF_N_TC_SetAudioStreamingIndicator_media_app_HMI_replies_TIMED_OUT.lua Check_that_SDL_responds_with_resultcode_TIMED_OUT_from_HMI_incase_SetAudioStreamingIndicator_is_correct
+./test_scripts/API/SetAudioStreamingIndicator/046_ATF_N_TC_SetAudioStreamingIndicator_media_app_HMI_replies_CHAR_LIMIT_EXCEEDED.lua Check_that_SDL_responds_with_resultcode_CHAR_LIMIT_EXCEEDED_from_HMI_incase_SetAudioStreamingIndicator_is_correct
+./test_scripts/API/SetAudioStreamingIndicator/047_ATF_N_TC_SetAudioStreamingIndicator_media_app_HMI_replies_INVALID_ID.lua Check_that_SDL_responds_with_resultcode_INVALID_ID_from_HMI_incase_SetAudioStreamingIndicator_is_correct
+./test_scripts/API/SetAudioStreamingIndicator/048_ATF_N_TC_SetAudioStreamingIndicator_media_app_HMI_replies_DUPLICATE_NAME.lua Check_that_SDL_responds_with_resultcode_DUPLICATE_NAME_from_HMI_incase_SetAudioStreamingIndicator_is_correct
+./test_scripts/API/SetAudioStreamingIndicator/049_ATF_N_TC_SetAudioStreamingIndicator_media_app_HMI_replies_TRUNCATED_DATA.lua Check_that_SDL_responds_with_resultcode_TRUNCATED_DATA_from_HMI_incase_SetAudioStreamingIndicator_is_correct
+./test_scripts/API/SetAudioStreamingIndicator/050_ATF_P_TC_SetAudioStreamingIndicator_media_app_HMI_replies_WARNINGS.lua Check_that_SDL_responds_with_resultcode_WARNINGS_from_HMI_incase_SetAudioStreamingIndicator_is_correct
+./test_scripts/API/SetAudioStreamingIndicator/051_ATF_P_TC_SetAudioStreamingIndicator_media_app_HMI_replies_WRONG_LANGUAGE.lua Check_that_SDL_responds_with_resultcode_WRONG_LANGUAGE_from_HMI_incase_SetAudioStreamingIndicator_is_correct
+./test_scripts/API/SetAudioStreamingIndicator/052_ATF_P_TC_SetAudioStreamingIndicator_media_app_HMI_replies_RETRY.lua Check_that_SDL_responds_with_resultcode_RETRY_from_HMI_incase_SetAudioStreamingIndicator_is_correct
+./test_scripts/API/SetAudioStreamingIndicator/053_ATF_P_TC_SetAudioStreamingIndicator_media_app_HMI_replies_SAVED.lua Check_that_SDL_responds_with_resultcode_SAVED_from_HMI_incase_SetAudioStreamingIndicator_is_correct
+./test_scripts/API/SetAudioStreamingIndicator/054_ATF_P_TC_SetAudioStreamingIndicator_media_app_HMI_replies_UNSUPPORTED_RESOURCE.lua Check_that_SDL_responds_with_resultcode_UNSUPPORTED_RESOURCE_from_HMI_incase_SetAudioStreamingIndicator_is_correct
```

Comment:
There are 2 result codes that are not covered: 'NO_APPS_REGISTERED' and 'NO_DEVICES_CONNECTED'
Please add test cases for them